webpack/assets/javascripts/react_app/common/MountingService.js

```diff
@@ -7,21 +7,49 @@ import componentRegistry from '../components/componentRegistry';
 
 export { default as registerReducer } from '../redux/reducers/registerReducer';
 
-const mountedNodes = [];
+/**
+ * mounted-nodes cache
+ */
+let mountedNodes = [];
```

Comment:
Doesn't matter much, but i think this can remain `const` since the constant is the array pointer, not the array itself. Also, instead of the comment, why not just call the variable `mountedNodesCache`?

Reply:
I actually do change the pointer, see line 47.
👍  for mountedNodesCache